decentraland/unity-renderer
# feat: asset bundle material transition

## What does this PR change?

Add a custom fade-in effect for when Asset Bundles are load or reused

## How to test the changes?

1) Go to: https://play.decentraland.zone/?renderer-branch=chore/asset-bundle-material-transition
2) Walk around and see that the custom fade in effect is present

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
